src/pages/keys/KeysContent.tsx

```
@@ -785,6 +873,110 @@ export default function KeysContent() {
+                    )}
+
+                    {selectedKey.appIds.length === 0 ? (
+                      <div className="text-sm text-gray-500 p-4 bg-gray-50 rounded-lg text-center">
+                        등록된 앱 ID가 없습니다.
+                      </div>
+                    ) : (
+                      <div className="space-y-2">
+                        {selectedKey.appIds.map(
+                          (appId: string, index: number) => (
+                            <div
+                              key={index}
```

Biome (2.4.6)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)